torch_xla/debug/profiler.py

```diff
@@ -0,0 +1,67 @@
+from concurrent import futures
+import torch_xla
+import torch_xla.core.xla_env_vars as xenv
+from typing import Optional
+
+
+def start_server(port: int):
+  """Start a profiler server on the client side on provided port.
+
+  Users can then use the tensorboard profiler plugin
+  (https://github.com/tensorflow/profiler) or the
+  :func:`~torch_xla.debug.profiler.trace` as the client to request
+  a profiler from this server.
+
+  Args:
+    port (int): the port to start the profiler server on.
+  Returns:
+    A `ProfilerServer` instance that dictates the lifecycle of the profiler
+    server. If this object is garbage collected, the profiler server is
+    shut down.
+  """
+  return torch_xla._XLAC.profiler.start_server(port)
```

Comment:
I'd make this function a no-op if port is 0 or None or negative. And document that behavior in the `port` arg

Reply:
Added an exception section to the docstring.